collectors/feature/jira/src/main/java/com/capitalone/dashboard/model/FeatureCollector.java

```diff
@@ -12,42 +13,59 @@
  * @author KFK884
  */
 public class FeatureCollector extends Collector {
+	private long lastRefreshTime;
+
+	public long getLastRefreshTime() {
+		return lastRefreshTime;
+	}
+
+	public void setLastRefreshTime(long lastRefreshTime) {
+		this.lastRefreshTime = lastRefreshTime;
+	}
+
+	public JiraMode getMode() {
```

Comment:
@Sbrenthughes  jira mode is by default "Board"? how do we set it to "Team" (no setter)

Reply:
it is passed in `FeatureCollectorTask.java` in the `getCollector()` method.